src/Validator/PhoneNumberValidator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+declare(strict_types=1);
+
+namespace EoneoPay\Externals\Validator;
+
+use EoneoPay\Externals\Validator\Interfaces\PhoneNumberValidatorInterface;
+use libphonenumber\NumberParseException;
+use libphonenumber\PhoneNumber;
+use libphonenumber\PhoneNumberFormat;
+use libphonenumber\PhoneNumberUtil;
+
+/**
+ * @SuppressWarnings(PHPMD.StaticAccess) Libphonenumber construction is with static methods
+ */
+class PhoneNumberValidator implements PhoneNumberValidatorInterface
+{
+    /**
+     * @var string
+     */
+    private $defaultRegion;
+
+    /**
+     * @var \libphonenumber\PhoneNumberUtil
+     */
+    private $parser;
+
+    /**
+     * Constructor
+     *
+     * @param string $defaultRegion
+     */
+    public function __construct(?string $defaultRegion = null)
+    {
+        $this->defaultRegion = $defaultRegion ?? 'AU';
```

Comment:
Is it the default region, or just the region? Default region implies that it can be overriden when `parseNumber()` and stuff is overidden.

Reply:
It is the region to use when no + is present, if you want to use an NZ number it must start with the +/country code.

You can force every number to require it by not providing a default region, but i think for an australian app, defaulting to +61 if it isnt present makes sense.

Reply:
> It is the region to use when no + is present

Ok, so needs to say that.